math/minuit2/src/AnalyticalGradientCalculator.cxx

```diff
@@ -57,33 +56,33 @@ FunctionGradient AnalyticalGradientCalculator::operator()(const MinimumParameter
       MnAlgebraicVector vg2(par.Vec().size());
       MnAlgebraicVector vgstep(par.Vec().size());
       for(unsigned int i = 0; i < par.Vec().size(); i++) {
-         unsigned int ext = fTransformation.ExtOfInt(i);
-         if (fTransformation.Parameter(ext).HasLimits()) {
-            double int2ext_g2 = fTransformation.D2Int2Ext(i, par.Vec()(i));
-            double int2ext_gstep = fTransformation.GStepInt2Ext(i, par.Vec()(i));
-            vg2(i) = int2ext_g2 * g2[ext];
-            vgstep(i) = int2ext_gstep * gstep[ext];
-         } else {
-            vg2(i) = g2[ext];
-            vgstep(i) = gstep[ext];
-         }
+        unsigned int ext = fTransformation.ExtOfInt(i);
+        if(fTransformation.Parameter(ext).HasLimits()) {
+          //double dd = (fTransformation.Parameter(ext).Upper() - fTransformation.Parameter(ext).Lower())*0.5*cos(par.Vec()(i));
+          //       const ParameterTransformation * pt = fTransformation.transformation(ext);
+          //       double dd = pt->dInt2ext(par.Vec()(i), fTransformation.Parameter(ext).Lower(), fTransformation.Parameter(ext).Upper() );
+          double dd = fTransformation.DInt2Ext(i, par.Vec()(i));
+          v(i) = dd*grad[ext];
+        } else {
+          v(i) = grad[ext];
+        }
       }
 
-      return FunctionGradient(v, vg2, vgstep);
-   } else {
+      MnPrint print("AnalyticalGradientCalculator");
+      print.Debug("User given gradient in Minuit2", v);
+
       return FunctionGradient(v);
-   }
-}
+    }
 
-FunctionGradient AnalyticalGradientCalculator::operator()(const MinimumParameters &par, const FunctionGradient &) const {
-   // needed from base class
-   return (*this)(par);
-}
+    FunctionGradient AnalyticalGradientCalculator::operator()(const MinimumParameters& par, const FunctionGradient&) const {
+      // needed from base class
+      return (*this)(par);
+    }
 
-bool AnalyticalGradientCalculator::CheckGradient() const {
-   // check to be sure FCN implements analytical gradient
-   return fGradCalc.CheckGradient();
-}
+    bool AnalyticalGradientCalculator::CheckGradient() const {
+      // check to be sure FCN implements analytical gradient
+      return fGradCalc.CheckGradient();
+    }
 
-}  // namespace Minuit2
+  }  // namespace Minuit2
```

Comment:
I think that's a misconfiguration of an editor. If I'm not mistaken, ROOT doesn't indent inside namespaces.